composer.json

```diff
@@ -24,14 +24,16 @@
         "symfony/framework-bundle": "~2.3",
         "symfony/twig-bundle": "~2.3",
         "symfony/form": "~2.3",
-        "symfony/security-bundle": "~2.3"
+        "symfony/security-bundle": "~2.3",
+        "symfony/symfony": "2.7.*@dev"
```

Comment:
please don't out a requirement on ``symfony/symfony`` (especially  not on ``2.7.*``)